### PR TITLE
PLT-10439: Add feature to disable access logs

### PR DESCRIPTION
**Pull request by tagomoris**
T/O

### CODE REVIEW

**Comment by tagomoris** [comment]
@komamitsu PTAL

File: src/main/java/com/treasuredata/underwrap/UnderwrapServer.java [hunk 1]
```diff
@@ -75,6 +76,11 @@ public UnderwrapServer(Class<? extends UnderwrapApplication> applicationClass, P
         this.serverRootPath = serverRootPath;
     }
 
+    public void disableAccessLog()
```
Comment (komamitsu) [inline src/main/java/com/treasuredata/underwrap/UnderwrapServer.java]: `setAccessLogEnabled` sounds a more common name to me.

Comment (tagomoris) [reply src/main/java/com/treasuredata/underwrap/UnderwrapServer.java]: I will update so.

**Comment by tagomoris** [comment]
@komamitsu Could you release the new version of Underwrap?

**Comment by komamitsu** [comment]
Sure. Released it as 0.1.7! Wait for a while until it gets visible.

**Comment by tagomoris** [comment]
Great 🎉

**Pull request merged**